ignite/cmd/scaffold_module.go

```diff
@@ -58,15 +58,21 @@ A module can depend on one or more other modules and import their keeper
 methods. To scaffold a module with a dependency use the "--dep" flag
 
 For example, your new custom module "foo" might have functionality that requires
-sending tokens between accounts. The method for sending tokens is a defined in
-the "bank"'s module keeper. You can scaffold a "foo" module with the dependency
-on "bank" with the following command:
+sending tokens between accounts. The method for sending tokens is defined in the
+bank's module keeper. You can scaffold a "foo" module with the dependency on
+"bank" with the following command:
 
   ignite scaffold module foo --dep bank
 
 You can then define which methods you want to import from the "bank" keeper in
 "expected_keepers.go".
 
+Provided with "--dep bank" Ignite looks up "BankKeeper" in "app.go". If
+"BankKeeper" doesn't exist in app.go, module scaffolding will fail. Sometimes
+you might want to specify the keeper name explicitly after a colon, for example:
```

Comment:
In my understanding, those 2 sentences speaks about 2 different things, what about using 2 different paragraphs ?

Reply:
I don't think they are about different things. It's about specifying dependencies and the colon syntax is essentially a hack to make importing modules like "feegrant" possible.

Reply:
I'm ok with having two paragraphs, not a big deal, I'm just trying to be concise and there might not be enough content for two paragraphs.

Reply:
Ok for me then, let's keep it like that :+1: